designs/administrator/olm/custom-operator-delete-message/index.md

```diff
@@ -0,0 +1,17 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Custom message on operator uninstall
+
+When an operator is uninstalled by an admin, the operator's developer may like to show a custom message in the uninstall dialog that might warn the admin of any consequences of uninstalling it, that might be particular to that operator.
+
+For example, when a storage operator is uninstalled, perhaps any associated data in that storage is also wiped, which the operator's developer would like to convey to the user.
+
+**Note:** The custom warning text used in this design is just for example-sake, and not actually intended to be use.
+
+## Uninstalling operator
+
+![Cluster dashboard status card message click](img/1-1-header.png)
```

Comment:
@dmesser @itsptk I know the sentence "Your application will keep running, but it will no longer receive updates or configuration changes." is what exists in the modal today, but what is the "application" that it's referring to exactly? And how does it keep running if I'm uninstalling it, or at least key component of it? Does this sentence mean the CRs the operator generated will stick around? How do I remove them later if that's the case?

Is there a clearer way of phrasing this for super noobs like me?

Reply:
Let me try a different version of this: "Removing the Operator will not remove any of its Custom Resource Definitions or managed resources. If your Operator deployed applications on this cluster or configured off-cluster resources these will continue to run and need to be cleaned up manually."

Reply:
Sounds clearer to me!

![1-1-header](https://user-images.githubusercontent.com/54860125/76972407-efab7b80-6904-11ea-8da3-426f51666c1c.png)